tasklist/hi.json

```
@@ -0,0 +1,520 @@
+{
+  " लेबल " : {
```

The 'labels' property is not a translate string. The translations will not be found by the Application if the top-level attributes are changed

```suggestion
  "labels" : {
```